hotsos/plugin_extensions/openstack/service_network_checks.py

```diff
@@ -69,6 +79,53 @@ def get_phy_port_health_info(self):
 
         return port_health_info
 
+    def get_gateway_router_mtus(self):
+        _project = getattr(self, 'neutron')
+        phy_mtus = set()
+        phy_ports = []
+        router_mtus = set()
+        if _project and _project.bind_interfaces:
+            for port in _project.bind_interfaces.values():
+                phy_ports.append({port.name: port.mtu})
+                phy_mtus.add(port.mtu)
+
+            for ns in self.cli.ip_netns():
+                # strip index
+                ns = ns.partition(" ")[0]
+                for nsprefix in ['qrouter', 'snat']:
+                    if not ns.startswith(nsprefix):
+                        continue
+
+                    for port in HostNetworkingHelper().get_ns_interfaces(ns):
+                        for pprefix in ['qr']:
```

Comment:
In the snat- namespaces there will be sg-* interfaces with this reduced mtu as well.

Reply:
good call. will fix